### PR TITLE
Add `tvos_app` example

### PR DESCRIPTION
**Pull request by cgrindel**
Related to #95.

- Added `tvos_app` example.
- Added `TvosXcTestBundleInfo` support to `targets.is_test_bundle()`.
- Added fixtures for `tvos_app`.

### CODE REVIEW

File: examples/tvos_app/Example/ContentView.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
+//
+//  ContentView.swift
+//  Example
+//
+//  Created by Chuck Grindel on 4/8/22.
+//
+
```
Comment (brentleyjones) [inline examples/tvos_app/Example/ContentView.swift:7]:
```suggestion
```

File: examples/tvos_app/Example/ExampleApp.swift [hunk 2]
```diff
@@ -0,0 +1,17 @@
+//
+//  ExampleApp.swift
+//  Example
+//
+//  Created by Chuck Grindel on 4/8/22.
+//
+
```
Comment (brentleyjones) [inline examples/tvos_app/Example/ExampleApp.swift:7]:
```suggestion
```

File: examples/tvos_app/ExampleUITests/ExampleUITests.swift [hunk 3]
```diff
@@ -0,0 +1,42 @@
+//
+//  ExampleUITests.swift
+//  ExampleUITests
+//
+//  Created by Chuck Grindel on 4/8/22.
+//
+
```
Comment (brentleyjones) [inline examples/tvos_app/ExampleUITests/ExampleUITests.swift:7]:
```suggestion
```

File: examples/tvos_app/ExampleUITests/ExampleUITestsLaunchTests.swift [hunk 4]
```diff
@@ -0,0 +1,32 @@
+//
+//  ExampleUITestsLaunchTests.swift
+//  ExampleUITests
+//
+//  Created by Chuck Grindel on 4/8/22.
+//
+
```
Comment (brentleyjones) [inline examples/tvos_app/ExampleUITests/ExampleUITestsLaunchTests.swift:7]:
```suggestion
```

**Comment by cgrindel** [comment]
@brentleyjones Sorry about the generated comments. I removed them.

File: examples/tvos_app/BUILD [hunk 5]
```diff
@@ -0,0 +1,9 @@
+load("//xcodeproj:xcodeproj.bzl", "xcodeproj")
+load(":xcodeproj_targets.bzl", "XCODEPROJ_TARGETS")
+
+xcodeproj(
+    name = "xcodeproj",
+    project_name = "TvOS App",
```
Comment (brentleyjones) [inline examples/tvos_app/BUILD:6]:
```suggestion
    project_name = "tvOS App",
```

**Review by brentleyjones — APPROVED** [review]

**Pull request merged**